ocs_ci/templates/ocs-deployment/stage-operator-source-secret.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: SECRET-NAME
```

Comment:
add -PLACEHOLDER as we discussed.

Same for others placeholders

Reply:
Fixed by e8b56d6.